models/org_team.go

```diff
@@ -145,15 +145,29 @@ func (t *Team) getUnits(e db.Engine) (err error) {
 
 // GetUnitNames returns the team units names
 func (t *Team) GetUnitNames() (res []string) {
+	if t.AccessMode >= perm.AccessModeAdmin {
+		return unit.AllUnitKeyNames()
+	}
+
 	for _, u := range t.Units {
 		res = append(res, unit.Units[u.Type].NameKey)
 	}
 	return
 }
 
-// HasWriteAccess returns true if team has at least write level access mode.
-func (t *Team) HasWriteAccess() bool {
-	return t.Authorize >= perm.AccessModeWrite
+// GetUnitsMap returns the team units permissions
+func (t *Team) GetUnitsMap() map[string]string {
+	var m = make(map[string]string)
```

Comment:
`var m =` -> `m :=`?

Reply:
why?

Reply:
Consistency. I don't think mixing `var name = value` and `name := value` is a good thing in the code. Style linters like `gofumpt` will rewrite these kind of lines into `name := ...`

Reply:
There is no mixing usages here(in this function). And I personally like `var` better. If Golang's code could pass `gofumpt`, I will not against it.